assets/css/style.css

```diff
@@ -74,6 +128,53 @@ button {
   background-repeat: no-repeat;
 }
 
+.ingredients-img {
+  padding-top: 70px;
+}
+
+.ingredients-item-2 {
+margin: 0px 30px 0px 30px
+}
+
+.ingredients-divider {
+  padding: 25px 0px 25px 0px;
+}
+
+.ingredients-section {
+  background-image: url(/assets/images/ingredients-bg.jpg);
+  background-position: 50% 0;
+  background-repeat: no-repeat;
+  display: flex;
+  justify-content: flex-end;
+  max-width: 1600px;
+  margin: 0 auto;
+}
+
+.ingredients-title {
+  color: #fff;
+  font-family: 'Yeseva One', 'Arial', 'Times New Roman', serif;
+  font-size: 36px;
+}
+
+.ingredients-txt {
+  color: #fff;
+  font-family: 'Lato', 'Arial', 'Times New Roman', sans-serif;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.ingredients-wrapper {
+  align-items: center;;
+  background-color: rgba(0, 0, 0, 0.67);
+  display: flex;
+  flex-direction: column;
+  max-width: 570px;
+  text-align: center;
+  padding: 82px 70px 66px 70px;
+  margin: 92px 215px 101px 0px;
```

Comment:
A u sure in such pixel perfect?

Reply:
should i change it?
like 80px 70px 65px 70px
90px 200px 100px 0px

Reply:
I need to see the deploy, but your suggestion is better then selected code